ext/mbstring/mbstring.c

```diff
@@ -2084,13 +2084,13 @@ static void handle_strpos_error(size_t error) {
 	case MBFL_ERROR_NOT_FOUND:
 		break;
 	case MBFL_ERROR_ENCODING:
-		php_error_docref(NULL, E_WARNING, "Unknown encoding or conversion error");
+		zend_value_error("Unknown encoding or conversion error");
```

Comment:
Let's leave this one as a warning. This error doesn't actually occur (as far as I know), but if it did, it would be due to invalid encoding in the input string, which shouldn't throw. You can change the error message to just "Conversion error" to make that clear. (Unknown encodings are caught separately.)